### DISCUSSION TITLE
Add GitHub contributions to profile

### DISCUSSION THREAD

**Discussion by 0x4248** [community/community#50420] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.104.0 (93753009)
Device: iPhone 12 (iOS 16.4)
</details>

I would like to see my GitHub contributions like on the web version

![image](https://user-images.githubusercontent.com/60709927/226171151-08f9eaa1-7ab1-4a4d-ab77-884ca0617921.png)

**Comment by Iron-Ham** [comment]
Hi @awesomelewis2007, thanks for getting in touch and the feature request! I'll bring it to our Product team so we can prioritize it accordingly.

**Reply by 0x4248** [reply]
Thank you for your reply and for raising it.

**Reply by Iron-Ham** [reply]
@awesomelewis2007 while perhaps not _exactly_ what you want (as it doesn't exist _within_ the app) -- we do have a Contribution Graph Widget for iOS Home Screens. 

![IMG_1A7D702B71B3-1](https://user-images.githubusercontent.com/3388381/226405577-3a9153e9-bbe5-4edf-a033-29e043876a70.jpeg)

**Reply by 0x4248** [reply]
> @awesomelewis2007 while perhaps not _exactly_ what you want (as it doesn't exist _within_ the app) -- we do have a Contribution Graph Widget for iOS Home Screens. 
> 
> 
> 
> ![IMG_1A7D702B71B3-1](https://user-images.githubusercontent.com/3388381/226405577-3a9153e9-bbe5-4edf-a033-29e043876a70.jpeg)
> 
> 

Yes I do know about this but I was  looking for a GitHub stats on the profile page and to see other people's stats.
